src/test/java/com/katelocate/menugenerator/RecipeRepositoryTests.java

```diff
@@ -4,8 +4,6 @@
 import com.katelocate.menugenerator.recipe.RecipeRepository;
 import com.katelocate.menugenerator.recipe.RecipeType;
 
-import io.restassured.RestAssured;
```

Comment:
It would be nice to add a reason why you decided to remove it in the commit message